esmvaltool/cmorizers/obs/cmor_config/cds_uerra.yml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
The `.yml` files are named exactly as the dataset, so please rename to `CDS-UERRA.yml`.